libevent fails to build in linux (WSL)

```
Package: libevent:x64-linux
Vcpkg version: 0.0.110-unknownhash
```

```
CMake Error at scripts/cmake/vcpkg_fixup_cmake_targets.cmake:42 (message):
  'vcpkg/packages/libevent_x64-linux/debug/cmake' does not exist.
Call Stack (most recent call first):
  ports/libevent/portfile.cmake:34 (vcpkg_fixup_cmake_targets)
  scripts/ports.cmake:72 (include)
```

поддтверждаю

```
-- Applying patch /root/tmp/vcpkg/ports/libevent/fix-target-files.patch
-- Applying patch /root/tmp/vcpkg/ports/libevent/fix-target-files.patch done
-- Configuring x64-linux-dbg
-- Configuring x64-linux-dbg done
-- Configuring x64-linux-rel
-- Configuring x64-linux-rel done
-- Build x64-linux-dbg
-- Build x64-linux-dbg done
-- Build x64-linux-rel
-- Build x64-linux-rel done
CMake Error at scripts/cmake/vcpkg_fixup_cmake_targets.cmake:42 (message):
  '/root/tmp/vcpkg/packages/libevent_x64-linux/debug/cmake' does not exist.
Call Stack (most recent call first):
  ports/libevent/portfile.cmake:34 (vcpkg_fixup_cmake_targets)
  scripts/ports.cmake:72 (include)


Error: Building package libevent:x64-linux failed with: BUILD_FAILED
```

--- a/ports/libevent/portfile.cmake
+++ b/ports/libevent/portfile.cmake
@@ -21,6 +21,7 @@ vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
     OPTIONS
+        -DEVENT_INSTALL_CMAKE_DIR:PATH=share/libevent
         -DEVENT__DISABLE_BENCHMARK=ON
         -DEVENT__DISABLE_TESTS=ON
         -DEVENT__DISABLE_REGRESS=ON
@@ -31,7 +32,7 @@ vcpkg_install_cmake()
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH cmake)
+vcpkg_fixup_cmake_targets()
 vcpkg_copy_pdbs()
 
 file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/libevent)

